### PR TITLE
Django 1.11

### LINKED ISSUE #106: Django 1.7 Upgrade
On September 2nd, Django 1.7 was officially released and we should upgrade at some point.  The big change that we will notice is doing away with South in favor of the new migration structure.  I did the port with Nadine and it went relatively smoothly.

### LINKED ISSUE #299: Update django to 1.11
It looks like the current django installation is insecure and no longer supported, and django 1.11 is a long term release.

### PR DESCRIPTION
**Pull request by jonathan-s**
So this should get django 1.11 up and running. It would be neat if @jessykate and/or @bfirsh can check this and see if the views seem to be working fine. 

Either way, I would suggest that once we have a staging or similar up and running that we continue to use Django 1.11 for the time being. 

In my opinion there is no need to upgrade to Django 2.0 very urgently. Django 1.11 has all the latest security fixes and long term support until 2020. (see: https://www.djangoproject.com/download/). I'd suggest that once we've used Django 1.11 in production for a while and it seems to be working fine. Only then should we move to Django 2.0. 

BUT: Django 2.2 will be released in the second? quarter of 2019. That's the next django release that will have long-term support. So modernomad should definitely jump on that as soon as possible. :) 

(closes #106 and closes #299)

### CODE REVIEW

**Comment by bfirsh** [comment]
@jonathan-s Needs a rebase!

**Comment by jonathan-s** [comment]
@bfirsh It's now been rebased!

File: gather/views.py [hunk 1]
```diff
@@ -9,16 +9,13 @@
...
-from django.template import RequestContext
 from django.contrib import messages
 from django.conf import settings
 from django.db.models import Q
 from gather.models import Event, EventAdminGroup, EventSeries
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
-from django.contrib import messages
 from django.views.decorators.csrf import csrf_exempt
 from django.template.loader import get_template
-from django.template import Context
```
Comment (bfirsh) [inline gather/views.py]: Heh. Running linters/static analysis over the whole codebase would probably be a productive way to clean it up...

**Comment by bfirsh** [comment]
Awesome! LGTM!

**Pull request merged**